PSSwagger/PSSwagger.psm1

```diff
@@ -935,6 +936,7 @@ function ConvertTo-CsharpCode
                 CodeCreatedByAzureGenerator                    = `$$codeCreatedByAzureGenerator
                 BootstrapConsent                               = `$$UserConsent
             }
+            `$env:PSModulePath += '$env:PSModulePath'
```

Comment:
>`$env:PSModulePath += '$env:PSModulePath' [](start = 12, length = 41)

This won't work as expected in all cases, like when PSCore is installed with PSSwaggerUtility module.

Reply:
The current PSModulePath contains PSCore modules. The only problem here will be that core tests probably use PSSwaggerUtility in the user/global PSCore module path instead of the dev path. However, using the PSCore module paths is preferred over the PS module paths in general. We can figure out how to get around this if we run into a problem.